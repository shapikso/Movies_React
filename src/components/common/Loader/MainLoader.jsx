import React from 'react';
import {StyledMainDiv} from './styled';
import Loader from "./Loader";

const MainLoader = () => (
    <StyledMainDiv>
        <Loader />
    </StyledMainDiv>
);

export default MainLoader;
