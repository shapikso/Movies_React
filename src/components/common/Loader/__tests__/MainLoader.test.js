import React from 'react';
import {shallow, mount} from 'enzyme';
import MainLoader from "../MainLoader";

describe('OverviewDetails', () => {

    it('should render correctly', () => {
        const component = shallow(<MainLoader/>);
        expect(component).toMatchSnapshot();
    });
    it('should render 6 div( all loader) ', () => {
        const component = mount(<MainLoader/>);
        expect(component.find('div')).toHaveLength(6);
    });
});