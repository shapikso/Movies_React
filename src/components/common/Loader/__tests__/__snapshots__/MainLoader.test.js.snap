// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLoader test should render correctly 1`] = `ShallowWrapper {}`;

exports[`OverviewDetails should render correctly 1`] = `ShallowWrapper {}`;
